imap_processing/glows/tests/test_glows_decom.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from imap_processing.glows.l0 import glows_decom
+
+logging.basicConfig(level=logging.ERROR)
+
+
+@pytest.fixture()
+def decom_test_data():
+    """Read test data from file"""
+    current_directory = Path(__file__).parent
+    packet_path = f"{current_directory}/glows_test_packet_20230920_v00.pkts"
+    data_packet_list = glows_decom.decom_packets(packet_path)
+    return data_packet_list
+
+
+def test_glows_decom_count(decom_test_data):
+    expected_hist_len = 61
+    expected_de_len = 311
+
+    assert len(decom_test_data) == 2
+
+    # Histogram data
+    assert len(decom_test_data[0]) == expected_hist_len
+
+    # Direct events data
+    assert len(decom_test_data[1]) == expected_de_len
+
+
+def test_glows_hist_data(decom_test_data):
+    for hist in decom_test_data[0]:
+        for key in hist.data:
+            assert hist.data[key] is not None
+        assert len(hist.data["HISTOGRAM_DATA"]) == 3599
+        assert min(hist.data["HISTOGRAM_DATA"]) > 10
+        assert max(hist.data["HISTOGRAM_DATA"]) < 170
```

Comment:
You made that awesome-looking plot, do you have the data output from that chart which you could turn into an array to assert against directly rather than the min/max?

Reply:
No, I don't. Ultimately, the plan I have discussed with the GLOWS team is to have some kind of statistical test for checking the output - so I didn't go too overkill with the testing now, because I plan to go back and add that once they provide that data.